postgres/manifest.json

```diff
@@ -15,8 +15,10 @@
   "version": "1.2.0",
   "guid": "e9ca29d5-5b4f-4478-8989-20d89afda0c9",
   "public_title": "Datadog-Postgres SQL Integration",
-  "categories":["datastore"],
+  "categories":["data store"],
```

Comment:
Why don't we add that category to other data stores (mongo, sqlserver, ...) ?

Reply:
we indeed should, for now it's just a quick transcription of an internal categorisation that we need to enhance